mlir/test/IR/greedy-pattern-rewrite-driver-top-down.mlir

```diff
@@ -0,0 +1,60 @@
+// RUN: mlir-opt %s -test-patterns="max-iterations=1 top-down=true" \
+// RUN:     --split-input-file | FileCheck %s
+
+// Tests for https://github.com/llvm/llvm-project/issues/86765. Ensure
+// that operands of a dead op are added to the worklist even if the same value
+// appears multiple times as an operand.
+
+// -----
```

Comment:
nit: Doesn't really matter, but this `-----` is not needed and will cause `mlir-opt` to process an "empty" test (which has just the comment).

Reply:
I actually did this on purpose to separate the file-level comment from the first test case, but I realize that's not really necessary and may look weird. Removed.